Lex.dk

saattrupdan opened the pull request:
Tiny script that loads the Lex.dk from a JSONL file.

I set the `date` feature to simply have the `string` data type, as the time zones in the dataset are variable, and PyArrow only supports dates with the same timezone. The dates can easily be converted to `datetime` objects using the `datetime.fromisoformat` function.

KennethEnevoldsen commented on dfm/data/lexdk/lexdk.py:
```diff
@@ -0,0 +1,31 @@
+    return Dataset.from_json(path, features=features)
+
+
+if __name__ == '__main__':
+    dataset = load_lexdk('lexdk.jsonl')
+    breakpoint()
```
Maybe just add this as a test? But without running it using:
```
@pytest.mark.skip(reason="Dataset not publicly available")
```

saattrupdan replied:
Added a test now 👌

KennethEnevoldsen reviewed (commented):
Looks good, would add a test as well, otherwise, nothing else to add.

MalteHB commented:
Looks great @saattrupdan! And nice addition with the test. It seems that it is not black formatted?

saattrupdan commented:
Added a test now, moved the loading script to `load.py` (which looks for the dataset in the `dfm/data/lexdk` folder), and updated the number of tokens in the dataset in the readme.

saattrupdan commented:
> Looks great @saattrupdan! And nice addition with the test. It seems that it is not black formatted?

Fixed now 👌

MalteHB reviewed (commented):
Good job!
